ci-operator/jobs/openshift/library-go/openshift-library-go-master-presubmits.yaml

```diff
@@ -193,3 +193,75 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )verify-deps,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    context: ci/prow/e2e-aws-encryption
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-library-go-master-e2e-aws-encryption
+    rerun_command: /test e2e-aws-encryption
+    run_if_changed: ^(pkg/operator/encryption)|^(test/e2e-encryption)
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --branch=master
+        - --give-pr-author-access-to-namespace=true
+        - --org=openshift
+        - --repo=library-go
+        - --resolver-address=http://ci-operator-configresolver
+        - --secret-dir=/usr/local/e2e-aws-encryption-cluster-profile
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=e2e-aws-encryption
+        - --template=/usr/local/e2e-aws-encryption
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_TYPE
+          value: aws
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-library-go-master.yaml
+              name: ci-operator-master-configs
+        - name: JOB_NAME_SAFE
+          value: e2e-aws-encryption
+        - name: TEST_COMMAND
+          value: |
+            mkdir -p /tmp/artifacts
+            make test-e2e-encryption JUNITFILE=/tmp/artifacts/unit_report.xml
```

Comment:
is this `unit_report.xml` or `e2e_report.xml` ?

Reply:
I think the pattern with others and where CI expects it is unit_report.xml